src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -509,6 +528,21 @@ const OrganizationStream = createReactClass({
     });
   },
 
+  renderAwaitingEvents(projects) {
+    let {organization} = this.props;
+    let project = projects.length === 1 ? projects[0] : null;
```

Comment:
So if "all projects" are selected we pass `null`, what does `ErrorRobot` do when `project` is `null`?

Is using the first project the pattern we're using for now?

Reply:
I wasn't able to get the selected projects to be `null`. I always got either an empty array or one populated with the selected projects.

The organization issue list gets pushed the current selection through its props. It should inherit the first project pattern that we're using. This code is aiming to guard against the situation where an org has access to multi-project selection and selects multiple projects with no events.